src/main/resources/liquibase/202307051132-rename-instance_id-to-host_id.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202307051132-1" author="khowell" dbms="postgresql">
+        <comment>Rename instance_measurements.instance_id to host_id and support both.</comment>
+        <renameColumn tableName="instance_measurements" oldColumnName="instance_id" newColumnName="host_id"/>
+        <sql>
+            alter table instance_measurements add column instance_id uuid generated always as (host_id) stored
+        </sql>
+        <rollback>
+            <sql>
+                alter table instance_measurements drop column instance_id
+            </sql>
+            <renameColumn tableName="instance_measurements" oldColumnName="host_id" newColumnName="instance_id"/>
+        </rollback>
+    </changeSet>
+    <changeSet id="202307051132-1" author="khowell" dbms="!postgresql">
```

Comment:
Would you mind adding a comment here noting that a) this changeset is run during tests and why we do it this way for hsqldb.  It took me a solid minute to try and figure out why only two changesets actually ran in the DB when there were 4 defined in the file.

Reply:
Done, see updated file when you get a chance